extensions/amp-consent/0.1/cmps.js

```diff
@@ -34,6 +34,12 @@ if (getMode().test || getMode().localDev) {
     'promptUISrc':
       '/examples/amp-consent/diy-consent.html?cid=CLIENT_ID&pid=PAGE_VIEW_ID',
   };
+
+  CMP_CONFIG['_test_'] = {
+    'consentInstanceId': '_test_',
+    'checkConsentHref': '/get-consent-v1?cid=CLIENT_ID&pid=PAGE_VIEW_ID',
+    'promptUISrc': '/test/fixtures/served/cmp-iframe.html',
```

Comment:
could we override the `promptUISrc` with inline config, and get rid of either `_ping_` or `_test_`

Reply:
Ahh yes, good point.